astroid/nodes/node_classes.py

```diff
@@ -859,33 +859,33 @@ def find_argname(self, argname, rec=False):
         return None, None
 
     def get_children(self):
```

Comment:
See other comment. If we could find a way where we only impact `__str__` and `__repr__` I would be very much in favour of that.